### ISSUE TITLE
feat req: Make  #[enumscribe(case_insensitive)] work on container level

### ISSUE THREAD

**Issue by lcmgh** [Pantonshire/enumscribe#1]
This does not make the `Test` field deserialization case insensitive:

```rust
#[derive(EnumSerialize, EnumDeserialize, PartialEq, Eq, Clone, Copy, Debug, Display)]
#[enumscribe(case_insensitive)]
pub enum Stage {
    Test,
}
```

**Comment by pantonshire** [comment]
Thank you for using `enumscribe`!

I have published version 0.2.0, which implements this feature request:

```rust
#[derive(EnumSerialize, EnumDeserialize, PartialEq, Eq, Clone, Copy, Debug)]
#[enumscribe(case_insensitive)]
pub enum Stage {
    Test,
}

assert_eq!(
    serde_json::from_str::<Stage>(r#""tEsT""#).unwrap(),
    Stage::Test
);
```

**Comment by lcmgh** [comment]
Wow - thanks so much! This lib is really useful as `strum` doesn't provide Serde serialization/deserialization.
